sys/linux/bpf.txt

```diff
@@ -78,6 +78,7 @@ bpf_map_lookup_arg {
 	map	fd_bpf_map
 	key	ptr64[in, array[int8]]
 	val	ptr64[out, array[int8]]
+	flags	flags[bpf_map_flags, int64]
```

Comment:
This can only contain BPF_F_LOCK, right?
If so we could make it a bit more precise because bpf_map_flags contain 4 values.

Reply:
Ah, right. I'll push an update.

Reply:
Pushed.